app/src/main/java/com/example/travellens/CameraActivity.java

```diff
@@ -68,31 +69,24 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
                 bitmap = ImageDecoder.decodeBitmap(source);
                 ByteArrayOutputStream bytes = new ByteArrayOutputStream();
                 bitmap.compress(Bitmap.CompressFormat.JPEG, 40, bytes);
-                File resizedFile = camera.getPhotoFileUri(photoFileName + "_resized", getApplicationContext());
+                File resizedFile = camera.getPhotoFileUri(PHOTO_FILE_NAME + "_resized", getApplicationContext());
                 try {
                     resizedFile.createNewFile();
-                    FileOutputStream fos = null;
-                    try {
-                        /*todo : You should use try-with-resources for resources like this so that they will be closed/cleaned up automatically.
-                            With the current logic, the fos won't be closed if there is an exception since there is no finally block that closes it.*/
-                        fos = new FileOutputStream(resizedFile);
-                        // Write the bytes of the bitmap to file
+                    try (FileOutputStream fos = new FileOutputStream(resizedFile)) {
                         fos.write(bytes.toByteArray());
-                        fos.close();
                     } catch (FileNotFoundException e) {
-                        Log.e("CAMERA ACTIVITY" , e.toString());
+                        Log.e(TAG , e.toString());
```

Comment:
Usually you want to pass the the exception object so that the full stacktrace is logged. Passing e.toString() will only log the exception message.

```suggestion
                        Log.e(TAG , e);
```